styles/style.css

```diff
@@ -132,6 +190,19 @@ button {
   margin-bottom: 0;
 }
 
+.article-top {
+  border: 2px solid #dccdc6;
+  display: flex;
+  justify-content: space-between;
+}
+
+.article_side-info {
+  width: 68%;
```

Comment:
Try and utilise css grid for this if possible, it can be a bit awkward doing it this way with specific percentage width

Reply:
Yeah it was abit of struggle doing it this way, but Im also struggling abit with grid too, I need more patience to use grid, I will keep trying 🙂